.github/workflows/on-pull-request-opened-and-synchronize.yml

```diff
@@ -137,17 +137,17 @@ jobs:
     steps:
       - uses: actions/checkout@v4
       - uses: ./.github/actions/pnpm
-      - run: pnpm dlx playwright@1.52.0 install --only-shell --with-deps chromium
+      - run: pnpm dlx playwright@1.52.0 install --no-shell --with-deps chromium
```

Comment:
[Configured](https://playwright.dev/docs/browsers#chromium-new-headless-mode) Playwright to use the new, higher fidelity [Chromium Headless](https://developer.chrome.com/blog/chrome-headless-shell) in tests. 

> The old Headless mode is a lightweight wrapper around Chromium's //content module, and it therefore has substantially fewer dependencies. 
> 
> New Headless on the other hand is the real Chrome browser, and is thus more authentic, reliable, and offers more features. This makes it more suitable for high-accuracy end-to-end web app testing or browser extension testing.